src/vstart.sh

```diff
@@ -293,6 +297,14 @@ case $1 in
     --cephadm )
         cephadm=1
         ;;
+    --cephadm-hosts )
+        cephadm_hosts="$2"
+        shift
+        ;;
+    --cephadm-ssh-config-path )
+        cephadm_ssh_config_path="$2"
+        shift
+        ;;
```

Comment:
what I don't like here is: vstart is already overloaded with **lots** of parameters already. Do you see any alternative?